11/slim-fonts/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openjdk:11.0.10-slim
```

Comment:
Could we use wanderadock/openjdk:11-slim as source image? It could then simplify the whole dockerfile to one from and one simple run command

Reply:
It will be ok if you add those deb packages to original Dockerfiles, please do not introduce new image type. Also, please sort and indent RUN arguments as in this example https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run 

And I hope you tested the image in advance (build and at least local usage...)

Reply:
Thanks, I've updated all 4 existing images. I've tested all of them (all built locally, used in GRE and then deployed to FI)